packages/react-router/modules/__tests__/useRouter-test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { MemoryRouter, Route, useRouter } from "react-router";
```

Comment:
To fix tests failing :)
```suggestion
import { MemoryRouter, Route, unstable_useRouter as useRouter } from "react-router";
```

Reply:
@mjackson Is it really that hard to change this and release it? :) Looks like you are deliberately waiting for React 16.7 to come out and stalling :)

Reply:
Please be patient, @FredyC. Thanks :)